ReactNativeTemplate/android/app/build.gradle

```diff
@@ -109,6 +109,11 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        debug {
+            defaultConfig {
+                targetSdkVersion 22
```

Comment:
Yes it is dev only" but the initial crush look really ugly And it actually cover several other issues that were revealed later. 
For unexperienced developers  is can be very confusing experience.

I can look into ReactActivity fix but it may take months before it goes thru